### PR TITLE
Updating .NET Files 

### PR DESCRIPTION
**Pull request by surbhig12**
### Change Description ###

As part of updating outdated contents for Tizen .NET files, updated the following files:

- create-dotnet-hybrid-package

- testing-your-app-on-gear

The files have been updated based ont he feedback given

### CODE REVIEW

**Comment by surbhig12** [comment]
This PR is referenced from PR #1109. 
@hakkyu-samsung, 
Please review the updated files here.

**Comment by surbhig12** [comment]
This PR is referenced from PR #1109.
@hakkyu-samsung,
A gentle reminder. Please review the updated files here.

**Comment by hakkyu-samsung** [comment]
> This PR is referenced from PR #1109.
> @hakkyu-samsung,
> A gentle reminder. Please review the updated files here.

Thanks for the reminder, I have approved the changes.

**Comment by surbhig12** [comment]
Thank you.

File: docs/application/dotnet/tutorials/testing-your-app-on-gear.md [hunk 1]
```diff
@@ -15,65 +15,66 @@ To debug an application on the Samsung Gear device:
...
+To install and execute your application on the Samsung Gear device over Wi-Fi, follow these steps:
 
-1. Open Visual Studio 2017 app on your developer computer.
+1. In the Visual Studio menu, select **Tizen** > **Tizen Device Manager** > **Remote Device Manager**.
 
-2. Select **Tizen** > **Tizen Device Manager** > **Remote Device Manager**.
+    ![Device Manager-Remote Device Manager](media/testing_your_app_on_gear8.png)
 
-![Device Manager-Remote Device Manager](media/testing_your_app_on_gear8.png)
+3. To search for remote Samsung Gear devices, click **Scan Devices**. A list of available remote devices appears.
```
Comment (mijin85cho) [inline docs/application/dotnet/tutorials/testing-your-app-on-gear.md:46]: Minor issue. Please change the list number 3. to 2.

File: docs/application/dotnet/tutorials/testing-your-app-on-gear.md [hunk 2]
```diff
@@ -15,65 +15,66 @@ To debug an application on the Samsung Gear device:
...
-3. Click **Scan Devices** to search for remote Samsung Gear devices. You can see a list of available remote devices.
+    ![Scan Device](media/testing_your_app_on_gear9.png)
 
-![Scan Device](media/testing_your_app_on_gear9.png)
-![Remote Device Manager](media/testing_your_app_on_gear10.png)
+    ![Remote Device Manager](media/testing_your_app_on_gear10.png)
 
-4. Select the IP of Samsung Gear Device that you want to connect to.
-![Remote Device Manager - Enable Connection](media/testing_your_app_on_gear11.png)
+4. Select the IP of the Samsung Gear device that you want to connect to.
```
Comment (mijin85cho) [inline docs/application/dotnet/tutorials/testing-your-app-on-gear.md:52]: Minor issue. Please change the list number 4. to 3.

File: docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md [hunk 3]
```diff
@@ -39,37 +40,39 @@ $[TIZEN_STUDIO_DIR]/tools/ide/bin$ ./tizen package -t wgt -s [CERTIFICATE_PROFIL
...
+		<icon src="icon.png"/>
+		<name>WorldClockWidget</name>
+		<tizen:privilege name="http://tizen.org/privilege/application.launch"/>
+		<tizen:profile name="wearable"/>
     </widget>
     ```
-    package id of web application is set like below :
-     __*package="TestHybrid"*__
+    > [!NOTE]
+	> Ensure that the package ID of the Web application is set to **TestHybrid**. For example, `package="TestHybrid"`.
```
Comment (mijin85cho) [inline docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md:65]: Suggestion. Isn't it better not to use "For example" here, as `package="TestHybrid"` is not an example but an actual string that should be in `package="TestHybrid"` form?

Same issue to line 95.

File: docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md [hunk 4]
```diff
@@ -39,37 +40,39 @@ $[TIZEN_STUDIO_DIR]/tools/ide/bin$ ./tizen package -t wgt -s [CERTIFICATE_PROFIL
...
 
-3. Create .NET Sample Application
+3. Launch the latest version of Visual Studio and create a .NET sample application.
 
     ![Create .NET App 1](./media/create_dotnet_sample_cs_1.png)
 
     ![Create .NET App 2](./media/create_dotnet_sample_cs_2.png)
```
[the remark this answers is not in the document]
Comment (surbhig12) [reply docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md:71]: @hakkyu-samsung ,

Request you to please provide the images as suggested by @mijin85cho.

@mijin85cho,
Thank you for the review. I will update the file after I get the images.

[the remark this answers is not in the document]
Comment (hakkyu-samsung) [reply docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md:71]: @mijin85cho, @surbhig12 
Thanks for the suggestions, I will upload the new images this week.

[the remark this answers is not in the document]
Comment (hakkyu-samsung) [reply docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md:71]: @surbhig12 
Providing the new images:
* https://d3unf4s5rp9dfh.cloudfront.net/Tizen_doc/create-csharp-project-1.png
* https://d3unf4s5rp9dfh.cloudfront.net/Tizen_doc/create-csharp-project-2.png
* https://d3unf4s5rp9dfh.cloudfront.net/Tizen_doc/create-csharp-project-3.png

File: docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md (hunk 3 above)
Comment (surbhig12) [reply docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md:65]: @mijin85cho,

The reason to use"For example" here is to explain the sentence in the note. How the package ID should be.

File: docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (surbhig12) [reply docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md:71]: @hakkyu-samsung

thank yuo for the images. Will update.

File: docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md (hunk 3 above)
Comment (mijin85cho) [reply docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md:65]: @surbhig12 I think this line is the additional description for line 55, the part of the sample code above.

image

As I don't think this line as an example explanation but the description to the above sample code, "For example" sounds weird to me. Why do we suddenly give example to the user while that code is already provided above?

Comment (surbhig12) [reply docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md:65]: @mijin85cho,
I understand your point. I will remove "For example.."

**Comment by surbhig12** [comment]
A gentle reminder. Please verify.

File: docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md [hunk 5]
```diff
@@ -88,10 +91,10 @@ $[TIZEN_STUDIO_DIR]/tools/ide/bin$ ./tizen package -t wgt -s [CERTIFICATE_PROFIL
     </manifest>  
     ```
 
-    package id of .NET application is set like below :
-    __package="org.tizen.example.CrossTemplate1.Tizen"__
+    > [!NOTE]
+	> Ensure that the package ID of the Web application is set to **org.tizen.example.CrossTemplate1.Tizen**.
```
Comment (mijin85cho) [inline docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md:95]: Question. In this section creating .NET application is described. 
So shouldn't the id is for .NET application? Is that package ID for the Web application?

**Original :** package id of _.NET application_ is set like below :
**Updated :** Ensure that the package ID of the _Web application_ is set to **org.tizen.example.CrossTemplate1.Tizen**.

File: docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md (hunk 3 above)
Comment (surbhig12) [reply docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md:65]: Fixed it. Thanks.

File: docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md (hunk 5 above)
Comment (surbhig12) [reply docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md:95]: Yes, you are right. Fixed it.

Comment (mijin85cho) [reply docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md]: Thanks for applying my comments @surbhig12 
However I'm still thinking of removed information from the original sentence.

Both line 65 and 95, in the description for the package id in the above sample code, "package=" is trimmed out. I'm not sure whether this trimming is good for the clarity.

Below are the changes in both lines.
line 65 :  
   __*package="TestHybrid"*__ => **TestHybrid**
line 95 : 
__package="org.tizen.example.CrossTemplate1.Tizen"__ => **org.tizen.example.CrossTemplate1.Tizen**

Would you please let me know your opinion?

Comment (surbhig12) [reply docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md]: @mijin85cho,

I too agree, in the code, it should be package id.

@hakkyu-samsung,
Please let us know your views as well.

Comment (hakkyu-samsung) [reply docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md]: @mijin85cho @surbhig12 
Sorry for the late response, I agree that the current word choices may confuse the readers on which element they should be looking for in the manifest files. I suggest we change the sentences number 2 and 4 of the numbered list to something like "Check the package id in the <manifest file name> file". Also, we should explicitly mention that the **package** attribute refers to the package id.

For example,

2. Check the application's package id in the **config.xml** file.

The **package** attribute of the **\<tizen:application\>** element refers to the application's package id, make sure that it is set to **TestHybrid**.

...

4. Check the application's package id in the **tizen-manifest.xml** file.

The **package** attribute of the **\<manifest\>** element refers to the application's package id, make sure that it is set to **org.tizen.example.CrossTemplate1.Tizen**.

Comment (surbhig12) [reply docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md]: @hakkyu-samsung,

Thank you so much for the detailed explanation. I have modified accordingly.

Comment (mijin85cho) [reply docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md]: Hello, @hakkyu-samsung 

Would you please check the latest version and let us know your opinion?

**Comment by surbhig12** [comment]
Thank you @mijin85cho and @hakkyu-samsung for the review.

**Pull request merged**